### DISCUSSION TITLE
git clone is slow

### DISCUSSION THREAD

**Discussion by gratefulEachOne** [dengxiaoning/cesium_dev_kit#34] [Q&A] [answered]
Why is git clone slow?
![图片](https://github.com/dengxiaoning/cesium_dev_kit/assets/168287691/1b56023a-485a-4997-86fe-390e661a33e0)

**Comment by dengxiaoning** [comment, marked as answer]
Sorry ,   it should be because the model file is too large, please wait ☕  a little longer when cloning ❗

**Reply by gratefulEachOne** [reply]
All right.
